### DISCUSSION TITLE
Q: Is it possible to delete/purge 'items-dryrun.sqlite3-wal'?

### DISCUSSION THREAD

**Discussion by wanageeska** [abraunegg/onedrive#2183] [Q&A] [answered]
Hi,
I've noticed that `items-dryrun.sqlite3-wal` is consuming 2.6GB on my disk. Is it possible to delete or purge it? What is it exactly? Some kind of cache? Will it purge itself once I synchronize for reals?
Thanks,
-w

**Comment by abraunegg** [comment, marked as answer]
@wanageeska 
The `items-dryrun.sqlite3-wal` is the runtime data file when using `--dry-run`

This file should be automatically removed once the application is exited or when `--dry-run` is not used.
